client/me/help/main.jsx

```diff
@@ -28,12 +28,21 @@ import HelpUnverifiedWarning from './help-unverified-warning';
 import { getUserPurchases, isFetchingUserPurchases } from 'state/purchases/selectors';
 import { PLAN_BUSINESS } from 'lib/plans/constants';
 import QueryUserPurchases from 'components/data/query-user-purchases';
+import config from 'config';
 
 /**
  * Module variables
  */
 const debug = debugModule( 'calypso:help-search' );
 
+const getSupportLocale = () => {
+	const localeSlug = getLocaleSlug();
+	if ( config( 'support_locales' ).indexOf( getLocaleSlug() ) > -1 ) {
```

Comment:
Since we already have the slug stored in `localeSlug`, let's just reuse it here :)
```
if ( config( 'support_locales' ).indexOf( localeSlug ) > -1 )
```

Reply:
Nice spotting. :)